public/sass/pages/_history.scss

```diff
@@ -0,0 +1,268 @@
+// History Table
+.history-table {
+  // .gf-form overrides
+  .gf-form-label { display: none; }
+
+  .gf-form-switch {
+    margin-bottom: 0;
+
+    input + label {
+      height: 3.6rem;
+      width: 110%;
+    }
+
+    input + label::before, input + label::after {
```

Comment:
all these overrides classes is not a good & maintainable way to write csss. If you need overrides make them modifiers to the original class, never override styles in a different component like this, it's the path to spaghetti css :) 

By class modifiers, I mean something like this:https://github.com/grafana/grafana/blob/master/public/sass/components/_gf-form.scss#L12

Overrrides other styles like what is done all over this is to be avoided, will try to clean it up